modules/quant_purger/src/Plugin/Purge/Queuer/QuantPurger.php

```diff
@@ -82,15 +82,9 @@ public function invalidateTags(array $tags) {
     if (!$this->initialize()) {
       return;
     }
-
-    $config = \Drupal::config('quant_purger.settings');
-
-    $blocklist = is_array($config->get('tag_blocklist')) ? $config->get('tag_blocklist') : [];
-
+    
     // Clean up the tags that we care about for this run.
     $tags = array_diff($tags, $this->invalidatedTags);
-    $tags = array_diff($tags, $blocklist);
```

Comment:
Moved this to the traffic registry, if tags or paths are matched the registry will not be updated with the matched request and won't be included as a candidate here.

Reply:
I have tested and things appear to be working properly so marking resolved.